test/evergreen.yml

```diff
@@ -2080,14 +2080,24 @@ tasks:
   # End of cppsuite test tasks.
   # Start of csuite test tasks
   - name: csuite-tests-fast
-    tags: ["pull_request"]
+    tags: ["pull_request", "pull_request_no_sanitizer"]
```

Comment:
I had a few attempts at this change, and adding new tags was the only one that would work with Evergreen. 
Feel free to ping me if you have an alternate suggestion